internal/server/webserver/api.go

```diff
@@ -0,0 +1,96 @@
+package webserver
+
+import (
+	"io"
+	"log"
+	"net/http"
+	"os"
+	"path/filepath"
+	"strings"
+
+	"github.com/NHAS/reverse_ssh/internal/server/webserver/shellscripts"
+)
+
+func download(w http.ResponseWriter, req *http.Request) {
```

Comment:
If you're going to split out the build and download functionalities, please do this in another pull request. This pull request is to resolve/add #79, not do this

Reply:
Because before this, there is only one downloaded api, and there is no need for an independent compilation api